index.html

```diff
@@ -1,225 +1,320 @@
-            <h2 class="Title">Contact me</h2>
-            <p class="CONTACT_form">
-                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
-            </p>
-            <form class="form_container" action="https://formspree.io/f/mnqroaaw" method="POST">
-                <fieldset class="form_inputs">
-                    <input class="form_input" name="name" type="text" placeholder="Yeremias" maxlength="30" required />
-                    <input class="form_input" name="email" type="email" placeholder="natajayanj14@gmail.com" required />
-                    <textarea class="form_text_area" name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                    <button type="submit" class="btn_submit">Get It Touch</button>
-                </fieldset>
-            </form>
-        </section>
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (2).png" alt="CSS" /><span>CSS</span>
+          </li>
+          </div>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        </div>
+      </section>
+      <section class="contact_form">
+        <h2 class="Title">Contact me</h2>
+        <p class="CONTACT_form">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form
+          class="form_container"
+          action="https://formspree.io/f/mnqroaaw"
+          method="POST"
+        >
+          <fieldset class="form_inputs">
+            <input
+              class="form_input"
+              name="name"
+              type="text"
+              placeholder="Yeremias"
+              maxlength="30"
+              required
+            />
+            <input
+              class="form_input"
+              name="email"
+              type="email"
+              placeholder="natajayanj14@gmail.com"
+              required
+            />
+            <textarea
+              class="form_text_area"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="btn_submit">Get It Touch</button>
+          </fieldset>
+        </form>
```

Comment:
- [ ] Your contact me section also needs some adjustment. The text at the top of your form covers more space compare to what is on the Figma design. kindly take a look at both designs and adjust your design to produce what we have on the guiding template.
- [ ] Kindly take note of the position of the `Get in touch` button on the desktop. It is ought to be aligned to the center of the form. Kindly look into that too and fix it.

### Your Design
![your-contact-section](https://user-images.githubusercontent.com/26597432/200882630-6166ba98-6309-4a86-a527-8e853311b33a.png)

### Figma Design

![figma-contact-section](https://user-images.githubusercontent.com/26597432/200882678-86cff8f6-0442-4e73-87e5-1b562d9b7a2c.png)